src/Command/SubProfileCommand.php

```diff
@@ -404,9 +404,7 @@ protected function excludeSubComponents(array $parent, DrupalStyle $io) {
    * {@inheritdoc}
    */
   protected function execute(InputInterface $input, OutputInterface $output) {
-    $io = new DrupalStyle($input, $output);
-
-    if ($this->confirmGeneration($io)) {
+    if ($this->confirmOperation()) {
```

Comment:
Why was this changed?

Reply:
Because `confirmGeneration()` is deprecated and thus failing the deprecation scan. Happily, all the deprecated function does is call the new one, and it doesn't take the `$io` argument.